client/README.md

```diff
@@ -0,0 +1,31 @@
+# Web Faucet
+
+Web Client to access the faucet. Powered by Catpcha v3
+
+## Why?
+
+We don't have any easy to use Faucet, our two current options are to [access Matrix and contact a bot](https://wiki.polkadot.network/docs/learn-DOT#getting-tokens-on-the-rococo-testnet) or look for [faucet available inside the Ink! documentation](https://use.ink/faucet).
```

Comment:
```suggestion
To simplify the use the Faucet through a standalone webpage and open ability to share it via social networks & google search.
Two current options are to [access Matrix and contact a bot](https://wiki.polkadot.network/docs/learn-DOT#getting-tokens-on-the-rococo-testnet) or [Ink! documentation](https://use.ink/faucet).
```

Reply:
Thanks for the suggestion! Will apply it!